itests/integration_test.go

```diff
@@ -82,6 +83,17 @@ func (suite *ItestSuite) Test_SendTransaction() {
 	suite.Equal(suite.cfg.Accounts[3].Amount+1000000000, uint64(b.GetAvailable()))
 }
 
-func TestItestSuite(t *testing.T) {
+func TestItest1Suite(t *testing.T) {
+	t.Parallel()
+	suite.Run(t, new(ItestSuite))
+}
+
+func TestItest2Suite(t *testing.T) {
+	t.Parallel()
+	suite.Run(t, new(ItestSuite))
+}
+
+func TestItest3Suite(t *testing.T) {
+	t.Parallel()
 	suite.Run(t, new(ItestSuite))
 }
```

Comment:
I think these tests can be replaced by one test with subtests. Take a look:
```go
func TestItestParallel(t *testing.T) {
	for _, name := range []string{"1", "2", "3"} {
		t.Run(name, func(t *testing.T) {
			t.Parallel()
			suite.Run(t, new(ItestSuite))
		})
	}
}
```

Reply:
Human readability is not good here

Reply:
Slash appears in the name of the container and the docker doesn't work again